src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -67,8 +74,11 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   private _required: boolean = false;
   private _disableRipple: boolean = false;
 
-  // Needs to be public to support AOT compilation (as host binding).
-  _hasFocus: boolean = false;
+  /** Reference to the focus state ripple. */
+  private _focusRipple: RippleRef;
```

Comment:
I really prefer `_focusRipple` here because it's just really clear that it's a ripple. On the checkbox we have it similar. Just `focusedRipple`.